code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -316,3 +316,35 @@
 	desc = "Whose smart idea was it to use toys as crowd control? Ages 18 and up."
 	projectile_type = /obj/item/projectile/bullet/reusable/foam_dart/riot
 	icon_state = "foamdart_riot"
+
+//Crossbow bolts
+
+/obj/item/ammo_casing/caseless/crossbowbolt/apfsdsbolt
```

Comment:
This probably means something but it looks like gibberish, give it a better name.